ping360_sonar/SonarMode mode
---
bool success